war/src/main/scss/abstracts/_theme.scss

```diff
@@ -199,6 +199,19 @@ $semantics: (
   --link-text-decoration--active: underline;
   --link-font-weight: 500;
 
+  // Command Palette
+  --command-palette-results-backdrop-filter: contrast(0.7) brightness(1.5)
```

Comment:
this looks lighter in dark theme than it does in the screenshots, is it intended to be this light?

<img width="848" alt="image" src="https://github.com/jenkinsci/jenkins/assets/21194782/fe730125-711e-4918-9e85-579908c38a92">

Reply:
<img width="678" alt="image" src="https://github.com/jenkinsci/jenkins/assets/43062514/51b209a1-58c9-49ed-b1c8-e7cecc7b9bda">

Should be using the colours from https://github.com/jenkinsci/dark-theme-plugin/pull/494

Reply:
Right I had an old local build of dark theme, fixed now